<launch>
  <!-- lenti_mark node -->
  <node name="lenti_mark" pkg="lenti_mark" type="lenti_mark" output="screen" >
    <!-- marker param file -->
    <param name="mk_param_file" value="$(find lenti_mark)/Data/MarkerParams/markerPara.yml"/>
    <!-- camera param -->
    <!-- type  0:file 1:topic (CameraInfo) -->
    <param name="campara_type" value="0"/>
    <!-- camera param file -->
    <param name="cam_param_file" value="$(find lenti_mark)/Data/CameraParams/Logicool640480.txt"/>
    <!-- debug (RViz) -->
    <!--TF frame_id world -->
    <param name="world_flg" value="true" />
    <!-- camera position -->
    <param name="cam_pos_x" value="0.0" />
    <param name="cam_pos_y" value="0.0" />
    <param name="cam_pos_z" value="0.5" />
  </node>
</launch>
